src/polus/plugins/_plugins/io.py

```diff
@@ -1,9 +1,11 @@
+# type: ignore
 """Plugins I/O utilities."""
 import enum
 import logging
 import pathlib
 import re
 import typing
+from functools import singledispatch
```

Comment:
I like that. It is a bit verbose, but enable some function overloading in pyhton